ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,198 @@
+---
+eip: -
```

Comment:
```suggestion
eip: 7737
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.